Resources/Prototypes/Entities/Structures/Machines/nuke.yml

```
@@ -111,8 +111,8 @@
       whitelist:
         components:
         - NukeDisk
-    alertLevelOnActivate: delta
-    alertLevelOnDeactivate: green
+    alertLevelOnActivate: scpDelta # Fire edit
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)